main.c

```diff
@@ -211,7 +210,9 @@ int main(int argc, char** argv) {
   srand((unsigned int)time(NULL));
 
   ddhcp_config config;
-  config.block_size = 32;
+  unsigned int block_size_pow = 5;
+
+  config.block_size = 1 << block_size_pow;
```

Comment:
Do we need to initialize `block_size` here at all? Is it used anywhere before l.423?

Reply:
A quick scan through the source showed no places where this field seems to be read in the affected part of the code. Not tested, but it should be safe to drop that assignment in line 215.